tests/lib/cli.js

```diff
@@ -1113,9 +1113,9 @@ describe("cli", () => {
     });
 
     describe("when passing --info", () => {
-        it("should print out debugging information concerning the local environment", async() => {
-            await cli.execute("--info");
-            assert.isTrue(log.info.calledTwice);
+        it("should print out debugging information concerning the local environment", () => {
+            cli.execute("--info");
+            assert.isTrue(log.info.called);
```

Comment:
It seems that `log.info()` is fired almost 4 times. But verifying it with the test suit doesn't seem to work :thinking: 
```js
assert.strictEqual(log.info.callCount, 4);
```